mne/io/meas_info.py

```diff
@@ -253,7 +353,7 @@ def _check_helium_info(helium_info):
     return helium_info
 
 
-class Info(dict, MontageMixin):
+class Info(dict, MontageMixin, ContainsMixin):
```

Comment:
-1 on this. It's a backward incompatible change because previously `x in info` would call `dict.__contains__`, now it will call `ContainsMixin.__contains__` which is a very different behavior

Reply:
... oh wait that's probably not true due to MRO, if so, never mind!

Reply:
Like someone calling `'chs' in info`? I don't understand the use case, all the `info` keys are made at instantiation and never change ASAIK

Reply:
> Like someone calling 'chs' in info? I don't understand the use case

`info` can be modified, is different for different raw types, for raw/epochs/evoked, etc. I'm pretty sure I've used code like this before instead of, say, `raw.info.get(..., fallback)`. The behavior is *not* actually changed by this PR due to `dict` showing up earlier in the list anyway, so we're actually okay.